tests/messages_tests/test_mixins.py

```diff
@@ -11,4 +12,6 @@ def test_set_messages_success(self):
         author = {'name': 'John Doe', 'slug': 'success-msg'}
         add_url = reverse('add_success_msg')
         req = self.client.post(add_url, author)
-        self.assertIn(ContactFormViewWithMsg.success_message % author, req.cookies['messages'].value)
+        # This assumes that the message is stored in the cookie and not compressed
+        value = b64_decode(req.cookies['messages'].value.split(":")[0].encode()).decode()
```

Comment:
Should we document this method of accessing the message stored in the cookie? As a related point, are cookies accessed in the browser by Javascript by some Django users?

Reply:
Hard to tell, for messages I'd generally say (hope) no.